MozillaFoundation/foundation.mozilla.org
# Homepage refresh new intro section

# Description

This PR updates the homepage model and view, adding a new section below the hero banner, with editable heading, contents, cta and page target. Repositions the `Focus areas` section into it and updates all styling to tailwind utility classes.

<img width="430" alt="image" src="https://github.com/user-attachments/assets/9004144f-0df6-4326-81ac-c65a67cc4c30">

*Mobile layout ATF*

<img width="431" alt="image" src="https://github.com/user-attachments/assets/732f7e96-7c81-4b93-8f3f-249105e3eb0c">

*Mobile layout BTF*

<img width="1204" alt="image" src="https://github.com/user-attachments/assets/9361b2cd-56a6-497b-9707-230701351786">

*Desktop layout*

<img width="954" alt="image" src="https://github.com/user-attachments/assets/93c3b8c2-c949-4d13-8b91-ad3f801f03dc">

*Wagtail page form UI*

# To test

1. Login to the [review app CMS](https://foundation-s-tp1-930-ho-hg3if5.herokuapp.com/cms/) with credentials `admin2:admin2`
2. [Update the homepage](https://foundation-s-tp1-930-ho-hg3if5.herokuapp.com/cms/pages/46/edit/) new section `hero bottom` fields, publish.
3. Review the [homepage](https://foundation-s-tp1-930-ho-hg3if5.herokuapp.com/en/) and confirm changes were reflected correctly.

Link to sample test page: https://foundation-s-tp1-930-ho-hg3if5.herokuapp.com/
Related PRs/issues: #12598

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-1209)
